tests/Api.Integration.Tests/Helpers/HttpClientTestExtensions.cs

```diff
@@ -0,0 +1,39 @@
+namespace Passwordless.Api.Integration.Tests.Helpers;
+
+public static class HttpClientTestExtensions
+{
+    public const string ApiKey = "test:public:2e728aa5986f4ba8b073a5b28a939795";
+    public const string ApiSecret = "test:secret:a679563b331846c79c20b114a4f56d02";
+
```

Comment:
These are the default keys from the Api used when seeding the database for local development.

Reply:
Is there merit to apply a similar approach as done here, so as to not rely on defaults in case they change?

https://github.com/bitwarden/passwordless-dotnet/blob/f37c27bfc50c65bc409aa5507fa1907a6fb9d269/tests/Passwordless.Tests.Infra/TestApi.cs#L85-L125

Or not worth the effort?

Reply:
I think its a good next iteration. Right now, its a little tied to the database seed script. But, we can add it later. I started down the path, and more and more things started to be added to make it work seamlessly. Its a good add, but let's get this in place.